provide-storage/backup-storage-provider/end-to-end-bsp-onboarding.md

```diff
@@ -0,0 +1,363 @@
+- Sufficient account balance for deposits and collateral
+- Stable network connection
+- Open network ports (30333, optionally 9944)
+
+### Hardware Requirements
+
+BSPs have similar hardware requirements to MSPs as they store backup data and must reliably submit proofs of storage.
+
+#### Specifications
+
+| Component | Requirement |
+|-----------|-------------|
+| **CPU** | 8 physical cores @ 3.4 GHz (Intel Ice Lake+ or AMD Zen3+) |
+| **RAM** | 32 GB DDR4 ECC |
+| **Storage (System)** | 500 GB NVMe SSD (chain data) |
+| **Storage (User Data)** | 1 TB NVMe SSD or HDD (minimum) |
+| **Network** | 500 Mbit/s symmetric |
+
+#### Important Considerations
+
+- **Separate storage volumes**: Keep chain data and user data on separate volumes for better I/O performance
+- **Storage expansion**: Plan for growth; user data storage should be easily expandable
+- **max-storage-capacity**: Set this CLI flag to **80% of available physical disk space** to leave headroom for filesystem overhead and temporary files
+- **Cloud compatible**: BSPs can run effectively on cloud VPS with dedicated storage volumes
+- **Proof submission**: Ensure reliable network connectivity for timely proof submissions
+
+### Deposit Requirements
+
+- **Base Deposit**: 100 MOCK (`SpMinDeposit`)
+- **Per GiB**: 2 MOCK (`DepositPerData`)
+- **Formula**: 100 + (`capacity_in_gib` × 2) + buffer
+
+Examples:
+
+- **800 GiB capacity**: 100 + (800 × 2) = 1,700 MOCK required (1,800 MOCK recommended)
+- **1.6 TiB capacity**: 100 + (1,638 × 2) = 3,376 MOCK required (3,500+ MOCK recommended)
+
+The deposit is held (reserved) from your account when you start the BSP registration process and remains held while you operate as a BSP. The deposit is returned when you deregister as a BSP.
+
+!!! note
+    Your BSP account must be funded before BSP registration
+
+## Project Structure
+
+This is how the project structure will look like, once everything is set up.
+
+```
+datahaven-bsp-node/
+├── datahaven-node
+├── bsp-data
+│   └── chains
+│   │   ├── datahaven_stagenet_local
+│   │   │   └── keystore
+│   │   └── datahaven_testnet
+│   │       └── keystore
+│   bsp-storage    
+│   └── storagehub    
+│       ├── file_storage
+│       └── forest_storage      
+├── Dockerfile
+├── docker-compose.yml
+└── bsp.log    
+```
+
+## Project Setup
+
+Before running a BSP node, you will need to obtain the `datahaven-node` client binary and the chain specifications for the network you want to join.
+
+### Download Latest Client Release
+
+From the [Releases](https://github.com/datahaven-xyz/datahaven/releases){target=\_blank} section of the DataHaven repo, the latest version of the `datahaven-node` binary can be found. Currently, the latest version is {{ networks.testnet.client_version }} and it can be downloaded directly from [this link](https://github.com/datahaven-xyz/datahaven/releases/download/{{ networks.testnet.client_version }}/datahaven-node).
```

Comment:
```suggestion
From the [Releases](https://github.com/datahaven-xyz/datahaven/releases){target=\_blank} section of the DataHaven repo, the latest version of the `datahaven-node` binary can be found. Currently, the latest version is {{ networks.testnet.client_version }} and it can be downloaded directly from [this link](https://github.com/datahaven-xyz/datahaven/releases/download/{{ networks.testnet.client_version }}/datahaven-node){target=_blank}.
```